wpimath/src/main/native/include/frc/geometry/Transform3d.h

```diff
@@ -33,6 +33,17 @@ class WPILIB_DLLEXPORT Transform3d {
    */
   Transform3d(Translation3d translation, Rotation3d rotation);
 
+  /**
+   * Constructs a transform with the given x, y, z and rotation components.
+   * 
+   * @param x The x component of the translation.
```

Comment:
Use Pose3d as a guide for the docs.

Reply:
The docs should look like this:
```cpp
  /**
   * Constructs a transform with x, y, and z translations instead of a separate
   * Translation3d.
   *
   * @param x The x component of the translational component of the transform.
   * @param y The y component of the translational component of the transform.
   * @param z The z component of the translational component of the transform.
   * @param rotation The rotational component of the transform.
   */
```